cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/cache/SqlCache.kt

```diff
@@ -79,8 +79,16 @@ class SqlCache(
 
   private var createdTables = ConcurrentSkipListSet<String>()
 
+  private val hexStrings: List<String>
+
   init {
     log.info("Configured for $name")
+
+    this.hexStrings = arrayListOf()
+    for (byte in Byte.MIN_VALUE..Byte.MAX_VALUE) {
+      val str = "%02x".format(byte.toByte())
+      this.hexStrings.add(str)
+    }
```

Comment:
Would you mind adding a comment here explaining what this is doing? It will look like voodo for anyone new to the codebase 😄

Reply:
👍 Fixed. Added a comment